lib/Driver.js

```diff
@@ -139,9 +138,21 @@ module.exports = class Driver {
    *
    * @returns {string} Returns the key's id.
    */
-  async computeKeyId({key}) {
+  computeKeyId({key}) {
     return `did:key:${key.fingerprint()}#${key.fingerprint()}`;
   }
+
+  /**
+   * Generates a public verification method from a keypair.
```

Comment:
Does "public" need to be here?

```suggestion
   * Generates a verification method from a key pair.
```

Reply:
Hmm... well, this is more like "Turn a keypair into a verification method w/o the private key information".